Add example for all admonitions (#309)
TYPO3-Documentation/TYPO3CMS-Guide-HowToDocument: Documentation/WritingReST/CheatSheet.rst

    text with a green box (as styled by our sphinx template).
    ---------------------------------------------------------------

> Visit :ref:`h2document:rest-admonitions` to see all allowed admonitions.

Reviewer: ```suggestion
Visit :ref:`h2document:rest-admonitions` to see the available admonitions.
```
